cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/ApplicationRequestsPlaner.java

```diff
@@ -38,9 +39,14 @@ private ApplicationRequestsPlaner(ApplicationsOperations appOperations, String a
 
     /**
      * Creates the request for CfNewObject
+     * 
      * @param newObject the CfNewObject to be visited
-     * @throws IllegalArgumentException if the newObject is neither an ApplicationBean or an ApplicationManifestBean
-     * @throws ApplyException that wraps the exceptions that can occur during the creation procedure.
+     * 
+     * @throws IllegalArgumentException if the newObject is neither an
+     *                                  ApplicationBean or an
+     *                                  ApplicationManifestBean
+     * @throws ApplyException           that wraps the exceptions that can occur
+     *                                  during the creation procedure.
      */
     @Override
     public void visitNewObject(CfNewObject newObject) {
```

Comment:
I think it would be a good idea to add a precondition check that asserts wether the request list is still empty. Just in case there is a bug.

Reply:
Yes, sounds reasonable, 
you should assert a "valid" object, which u can really handle and which has a valid content.